infrastructure/config/config_yaml.py

```diff
@@ -49,5 +55,23 @@ def __init__(self, filename: str) -> None:
             self._jitter = Thresholds(config["thresholds"]["jitter"])
             self._packet_loss = Thresholds(config["thresholds"]["packet_loss"])
             self._timeout = tuple(config["timeout"])
+            self._logging = self._parse_logging_level(config)
         except Exception as err:
             raise Exception("Configuration error") from err
+
+    def _parse_logging_level(self, config) -> int:
+        try:
+            if config["logging_level"] == "FATAL" or config["logging_level"] == "CRITICAL":
+                return logging.CRITICAL
+            if config["logging_level"] == "ERROR":
+                return logging.ERROR
+            if config["logging_level"] == "WARN" or config["logging_level"] == "WARNING":
+                return logging.WARNING
+            if config["logging_level"] == "INFO":
+                return logging.INFO
+            if config["logging_level"] == "DEBUG":
+                return logging.DEBUG
+            else:
+                return logging.NOTSET
```

Comment:
I think that having typo in config should lead to throwing error, because logging is important part of app.